### DISCUSSION TITLE
Do we need cmp-nvim-lua anymore after installing lazydev.nvim?

### DISCUSSION THREAD

**Discussion by ysl2** [folke/lazydev.nvim#36] [Q&A] [answered]
Hi, thanks for your works !

I find out if I installed lazydev.nvim, then even if I remove the cmp-nvim-lua source in nvim-cmp, cmp can also prompt lua things. 

So I wonder if we installed lazydev, then the cmp-nvim-lua is not need? Or not? Thanks :-)

**Comment by mawkler** [comment]
I'm wondering the same thing? I'm guessing that the answer is that you don't need cmp-nvim-lua? I started getting duplicate suggestions of all built-in Neovim functions, which I solved by removing cmp-nvim-lua

**Comment by mairs8** [comment]
is anyone able to opine on this?

**Reply by sid-6581** [reply]
I've never used or needed `cmp-nvim-lua`, and I've not had any issues with Lua completions.

**Reply by folke** [reply, marked as answer]
yeah, you don't need that. That's what an LSP server does
